src/ccsr/src/client.rs

```diff
@@ -179,6 +191,20 @@ impl Client {
         Ok(res.id)
     }
 
+    /// Sets the compatibility level for the specified subject.
+    pub async fn set_subject_compatibility_level(
+        &self,
+        subject: &str,
+        compatibility_level: &CompatibilityLevel,
```

Comment:
```suggestion
        compatibility_level: CompatibilityLevel,
```

Since you've made `CompatibilityLevel` copy, not much point to taking it by reference.